docs/development/astropy-package-template.rst

```diff
@@ -124,7 +127,7 @@ by ``CHANGES.md`` in the instructions.
    all pass, you can proceed on.
 
 #. If you did the previous step, do ``git clean -fxd`` again to remove anything
-   you made there.  Run ``python setup.py build sdist --format=gztar`` to
+   you made there.  Run ``python -m pep517.build --source .`` to
```

Comment:
This will need to be replaced by python-build in the short term (https://python-build.readthedocs.io/en/latest/). I'm not sure exactly what's the status of this, but `pep517.build` is going to be replaced by python-build soon.

Reply:
Until `python-build` is shipped officially everywhere, we cannot recommend it yet.

Reply:
Yes, there's a bit of urgency to make pep517.build disappear, see discussion https://github.com/pypa/pep517/pull/83

If this bit of documentation is going to stick around in a stable version for some months, I'd say it's risky

Reply:
You can install python-build already, so I don't think there is an issue with recommending it now.

Reply:
Agree we should mention python-build here

Reply:
But why? Isn't it too soon? I tried using it in one of my packages and it doesn't work.

Reply:
all python build is, is a CLI for pep517 it should work fine. What trouble did you run into?

Reply:
My trouble: https://dev.azure.com/spacetelescope/acstools/_build/results?buildId=3756&view=logs&j=b25f120e-e38e-5d50-120e-fa71f5e29986&t=b1d60ffd-f6c2-56b4-1446-ab1e65159582&l=71

Reply:
You didn't install the `build` package :wink: Also, have you considered using https://github.com/OpenAstronomy/azure-pipelines-templates I hear some really awesome people maintain it :stuck_out_tongue:

Reply:
Huh... You would think Python would install it automatically since it is like so essential. Thanks! I'll try that.

Re: Azure -- I would have just switched to Actions. I just don't have time to refactor the CI for that package. 😬